requirements.txt

```diff
@@ -1 +1,2 @@
-ovos_utils
\ No newline at end of file
+ovos_utils
+ovos_workshop~=0.0.10
```

Comment:
please use ~=0.0, >=0.0.10
seems to work better when handling requirements with pip installs